snap/snapcraft.yaml

```diff
@@ -51,6 +51,7 @@ apps:
       - dot-google
       - dot-kubernetes
       - dot-maas
+      - dot-minikube
```

Comment:
I wonder if we need to automatically configure this out of the box, there is an option to use  `snap connect juju:dot-minikube`. This could reduce the number of files we're potentially going to read out of the box.

Reply:
I don't think we have to automatically configure this, but in that case a message reminding users to run the command or documentations somewhere would be nice.